source/13_envs/evogym.rst

```diff
@@ -0,0 +1,226 @@
+Evogym
+~~~~~~~~~
+
+Overview
+=========
+
+Evolution Gym (Evogym) is the first large-scale benchmark for co-optimizing the design and control of soft robots.
+Each robot is composed of different types of voxels (e.g., soft, rigid, actuators), resulting in a modular and expressive robot design space.
+The environment spans a wide range of tasks, including locomotion on various types of terrains and manipulation.
```

Comment:
The environment spans a wide range of tasks, including locomotion and manipulation on various types of terrains.